tf2_ros/src/tf2_ros/buffer.py

```diff
@@ -218,6 +218,18 @@ def can_transform_full(self, target_frame, target_time, source_frame, source_tim
             return core_result
         return core_result[0]
 
+    def _wait_for_transform_async(self, target_Frame, source_frame, time, callback):
+        fut = rclpy.task.Future()
+        if self.can_transform_core(target_frame, source_frame, time)[0]:
+            # Short cut, the transform is available
+            fut.set_result(True)
```

Comment:
>On #194 you wrote:
>
>>    the result on future doesn't contain any info about the transform. It should probably have that.
>
>Can you please clarify what information you're hoping to propagate up?

I was trying to say it would be nice if `fut.result()` returned the transform that was being waited for instead of `True`.